Doc/c-api/time.rst

```diff
@@ -0,0 +1,103 @@
+.. highlight:: c
+
+PyTime C API
+============
+
+.. versionadded:: 3.13
+
+PyTime API
+----------
+
+The PyTime_t API is written to use timestamp and timeout values stored in
+various formats and to read clocks with a resolution of one nanosecond.
+
+The :c:type:`PyTime_t` type is signed to support negative timestamps. The
+supported range is around [-292.3 years; +292.3 years]. Using the Unix epoch
+(January 1st, 1970) as reference, the supported date range is around
+[1677-09-21; 2262-04-11].
+
+
+Types
+-----
+
+.. c:type:: PyTime_t
+
+   Timestamp type with subsecond precision: 64-bit signed integer.
+
+   This type can be used to store a duration. Indirectly, it can be used to
+   store a date relative to a reference date, such as the UNIX epoch.
+
+
+Constants
+---------
+
+.. c:var:: PyTime_t PyTime_MIN
+
+   Minimum value of the :c:type:`PyTime_t` type.
+
+   :c:var:`PyTime_MIN` nanoseconds is around -292.3 years.
+
+.. c:var:: PyTime_t PyTime_MAX
+
+   Maximum value of the :c:type:`PyTime_t` type.
+
+   :c:var:`PyTime_MAX` nanoseconds is around +292.3 years.
+
+
+Functions
+---------
+
+.. c:function:: double PyTime_AsSecondsDouble(PyTime_t t)
+
+   Convert a timestamp to a number of seconds as a C :c:expr:`double`.
+
+   The function cannot fail.
+
+
+.. c:function:: PyTime_t PyTime_Monotonic(void)
+
+   Get the monotonic clock: clock that cannot go backwards.
```

Comment:
I think the Python time API is a bit more understandable. "Get the monotonic clock" sounds like it might refer to getting the "clock" itself rather than the current time.

Maybe:

Return the current time (in nanoseconds) using a monotonic clock, i.e., a clock that cannot go backwards. The clock is not affected by system clock updates...